utils/message_utils.js

```diff
@@ -184,7 +184,7 @@ let MessageUtils = {
   loadMessagePackage(msgPackage) {
     const indexPath = messagePackagePathMap[msgPackage];
     if (indexPath === undefined) {
-      throw new Error('Unable to find message package %s', msgPackage);
+      throw new Error('Unable to find message package ' + msgPackage);
```

Comment:
Is there a JS style guide or linter dictating this change? If not, we should probably choose one. I assume this change is marginally better that it was previous?